nbpages/converter.py

```diff
@@ -320,6 +321,48 @@ def clean_keyword(kw):
     return kw.strip().title().replace('.', '').replace('/', '').replace(' ', '')
 
 
+def get_changed():
+    '''
+    Returns the names of new/changed notebooks on current commit.
+    Returns .* if any global or .circleci/*  files were changed.
+    Returns None if no files were changed on current commit.
+    '''
+
+    # List of any files that when changed should convert all notebooks
+    global_files = ['convert.py', 'environment.yml', 'index.tpl',
+                    'nb_html.tpl', 'pages.css', 'gh_pages_deploy.sh']
+
+    # This command will list all changed files on current commit from master.
+    list_changed_files_cmd = "git show --pretty=format: --name-only -r master"
```

Comment:
the use of ``master`` here is a bit problematic, as a user might want to instead look for changes against some other base branch.  So maybe just:
```suggestion
    list_changed_files_cmd = "git show --pretty=format: --name-only -r {}".format(base_branch)
```
and add ``base_branch`` as a keyword to the function that then defaults to ``master``?